pybop/parameters/parameter.py

```diff
@@ -372,6 +372,12 @@ def get_sigma0(self) -> list:
 
         return sigma0
 
+    def priors(self) -> list:
```

Comment:
Should this be a `JointPrior` instead (see suggestion below)

Reply:
Not sure if I follow on this one, are you suggesting it should be renamed `joint_priors` or something else?

Reply:
I think I have the same question as Nicola. Is (conceptually) this `priors` the same as the `JointPriors` later on or not necessarily?

Reply:
Gotcha, it's not necessarily a `JointPrior`, as conceptually it's a helper method to combine all the priors from the individual parameters into a list. This is used later within the `JointPrior`, but not exclusively.